playbooks/roles/custom_certificates/tasks/main.yml

```diff
@@ -87,3 +87,6 @@
       -out "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
   args:
     creates: "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
+
+- include: issue.yml
+  when: custom_certificates_hostname is defined
```

Comment:
--extra-vars in packer was interpreting "True" as a string, but like I said, I'll come back to this and fix it if supports arrays.